public/src/main/java/com/revenuecat/purchases/Package.kt

```diff
@@ -9,14 +9,21 @@ import kotlinx.parcelize.Parcelize
  * @property identifier Unique identifier for this package. Can be one a predefined package type or a custom one.
  * @property packageType Package type for the product. Will be one of [PackageType].
  * @property product [StoreProduct] assigned to this package.
- * @property offering offeringID this package was returned from.
+ * @property offering offering this package was returned from.
+ * @property subscriptionPeriod Google BC5 only. String representation of the subscription period, used for filtering ProductDetails
+ * for the proper base plan to create a StoreProduct. For now, we are not worrying if it matches PackageType
+ * @property storeProductIdentifier  BC4/Amazon sku, BC4 subscription ID. used for querying the store.
```

Comment:
I know these aren't the final docs, but just a thought here: I imagine most customers will not know what BC5 / BC4 stands for. Perhaps with the build flavors we can try to find a way to fork the docstrings so we have one for each billing client, since no app will have both at the same time. 
This was different with StoreKit 1 vs 2 because all apps have both at the same time